sway_libs/src/signed_integers/i128.sw

```diff
@@ -0,0 +1,175 @@
+library i128;
+
+use std::u128::U128;
+
+/// The 128-bit signed integer type.
+/// Represented as an underlying U128 value.
+/// Actual value is underlying value minus 2 ^ 127
+/// Max value is 2 ^ 127 - 1, min value is - 2 ^ 127
+pub struct I128 {
+    underlying: U128,
+}
+
+pub enum Error {
+    ZeroDivisor: (),
+}
+
+pub trait From {
+    /// Function for creating I128 from U128
+    fn from(underlying: U128) -> Self;
+}
+
+impl From for I128 {
+    /// Helper function to get a signed number from with an underlying
+    fn from(underlying: U128) -> Self {
+        Self { underlying }
+    }
+}
+
+impl core::ops::Eq for I128 {
```

Comment:
Is this also part of the dependency ordering issue as to why this is here and not below the `impl I128`? The same goes for line 35.

Reply:
Why should it be below the impl I128?